lib/config/presets/gitlab.ts

```diff
@@ -4,12 +4,11 @@ import { Preset } from './common';
 
 const { get: glGot } = api;
```

Comment:
Will the token from the config automatically be used for API calls?

Reply:
Yes, i think so.